services/app-graphql/src/schema.graphql

```diff
@@ -689,3 +711,144 @@ type FetchEmailSettingsPayload {
+    s3URL: String!
+    "The sha256 is a unique string representing the file, generated on the FE currently in the FileUpload component"
+    sha256:  String!
+}
+
+"""
+Rates are rate certifications and their associated actuary contacts and documents
+State users may create, update, and submit several rates at a time
+"""
+enum RateType {
+  NEW
+  AMENDMENT
+}
+
+enum RateCapitationType {
+  RATE_CELL
+  RATE_RANGE
+}
+
+
+enum ActuaryCommunication {
+  OACT_TO_ACTUARY
+  OACT_TO_STATE
+}
+
+enum ActuarialFirm {
+  MERCER
+  MILLIMAN
+  OPTUMAS
+  GUIDEHOUSE
+  DELOITTE
+  STATE_IN_HOUSE
+  OTHER
+}
+
+type ActuaryContact  {
+  name: String!
+  titleRole: String!
+  email: String!
+  actuarialFirm: ActuarialFirm!
+  actuarialFirmOther: String
+}
+
+type PackageWithSameRate  {
+  packageName: String!
+  packageId: String!
+}
+
+
+type RateFormData {
+    rateType: RateType
+    rateCapitationType: RateCapitationType
+    rateDocuments: [GenericDocument!]!
+    supportingDocuments: [GenericDocument!]!
+    rateDateStart: DateTime
+    rateDateEnd: DateTime
+    rateDateCertified: DateTime
+    amendmentEffectiveDateStart: DateTime
+    amendmentEffectiveDateEnd:  DateTime
+    rateProgramIDs: [String!]!
+    rateCertificationName: String
+    certifyingActuaryContacts: [ActuaryContact!]!
+    addtlActuaryContacts: [ActuaryContact!]!
+    actuaryCommunicationPreference: ActuaryCommunication
+    packagesWithSharedRateCerts: [PackageWithSameRate!]!
+}
+
+type RelatedContractRevisions {
+    id: String
+    # contractID: String  # this would be needed in order to link to submission summary
+    submitInfo: UpdateInformation
+    unlockInfo: UpdateInformation
+    createdAt: DateTime
+    updatedAt: DateTime
+}
+
+type RateRevision {
+  id: ID!
+  createdAt: DateTime!
+  updatedAt: DateTime!
+  """
+    Information about who, when, and why this revision was unlocked.
+    Will be blank on the initial revision.
+    """
+  unlockInfo: UpdateInformation
+    "Information on who, when, and why this revision was submitted."
+  submitInfo: UpdateInformation
+  formData: RateFormData!
+  contractRevisions: [RelatedContractRevisions!]!
+}
+
+type Rate {
+    id: ID!
+    createdAt: DateTime!
+    updatedAt: DateTime!
+    """
+    Where the package is in the submission flow.
+    SUBMITTED and RESUBMITTED packages cannot be modified
+    """
+    status: HealthPlanPackageStatus!
+    "The initial date this rate was submitted at. Is not changed by unlock or resubmission."
+    initiallySubmittedAt: Date
+    "The state code (e.g. CA or TN) for the submitting state"
+    stateCode: String!
+    "Fuller state data for the submitting state"
+    state: State!
+    stateNumber: Int!
+    """
+    Array of revisions for this rate. Each revision represents a single submission
+    for the rate and contains the full data from when the rate cert was submitted
+    """
+    revisions: [RateRevision!]!
+    contractRevisions:[RelatedContractRevisions!]!
```

Comment:
I'm not sure what contractRevisions means for a Rate. We have that linkage in the DB but only because of the contractRevision.draftRates linkage. I don't know what the front end will use Rate.contractRevisions for